fe/src/components/Navbar/NavigationBar.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <v-app-bar app>
-    <div class="title">SunBnB</div>
-    <input placeholder="Anywhere" class="search-bar" />
+    <div class="title"><a href="/">SunBnB</a></div>
+    <input placeholder="Anywhere" class="search-bar" v-model="searchInput" @keyup.enter="search" />
```

Comment:
you can enclose this properly in a form tag instead of checking the keyup

Reply:
changed it already maam